_posts/2016-11-21-elections.md

```diff
@@ -28,4 +28,4 @@ The selection of Board members is an opportunity for each of you to have a voice
 Benjamin Bohl 
 MEI Elections Officer 2016 by appointment of the MEI-Board 
 
-[1] The By-laws of the Music Encoding Initiative are available online at: [http://music-encoding.org/community/mei-organization/mei-by-laws/](http://music-encoding.org/community/mei-organization/mei-by-laws/)
\ No newline at end of file
+[1] The By-laws of the Music Encoding Initiative are available online at: [https://music-encoding.org/community/mei-by-laws.html](https://music-encoding.org/community/mei-by-laws.html)
```

Comment:
```suggestion
[1] The By-laws of the Music Encoding Initiative are available online at: [http://music-encoding.org/community/mei-organization/mei-by-laws/](https://music-encoding.org/community/mei-by-laws.html)
```

Reply:
Do not change the content of the post (displayed "label" of the link, only the link target itself)